File: 06/03/Tests/LoginTest.cs
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;

namespace Tests
{
    [TestFixture]
    class LoginTest
    {
        IWebDriver Driver;

        [SetUp]
        public void SetUp()
        {
            //FirefoxOptions Options = new FirefoxOptions() { UseLegacyImplementation = true };
            //Driver = new FirefoxDriver(Options);
            var VendorDirectory = System.IO.Directory.GetParent(
                                    System.AppDomain.CurrentDomain.BaseDirectory).
                                        Parent.Parent.FullName
                                            + @"\Vendor";
            var Service = FirefoxDriverService.CreateDefaultService(VendorDirectory);
            Driver = new FirefoxDriver(Service);
        }

        [TearDown]
        public void TearDown()
        {
            Driver.Quit();
        }

        [Test]
        public void ValidAccount()
        {
            Driver.Navigate().GoToUrl("http://the-internet.herokuapp.com/login");
            Driver.FindElement(By.Id("username")).SendKeys("tomsmith");
            Driver.FindElement(By.Id("password")).SendKeys("SuperSecretPassword!");
            Driver.FindElement(By.CssSelector("button")).Click();
            System.Threading.Thread.Sleep(1000);
            Assert.That(Driver.FindElement(By.CssSelector(".flash.success")).Displayed);
            //Assert.That(Driver.FindElement(By.CssSelector(".flash.successasdf")).Displayed);
        }
    }
}

File: 06/03/tests/login_test.py
import pytest
import os
from selenium import webdriver
from selenium.webdriver.common.by import By
import time


class TestLogin():

    @pytest.fixture
    def driver(self, request):
        #driver_ = webdriver.Firefox(capabilities={'marionette': False})
        _geckodriver = os.path.join(os.getcwd(), 'vendor', 'geckodriver')
        driver_ = webdriver.Firefox(executable_path=_geckodriver)

        def quit():
            driver_.quit()

        request.addfinalizer(quit)
        return driver_


    def test_valid_credentials(self, driver):
        driver.get("http://the-internet.herokuapp.com/login")
        driver.find_element(By.ID, "username").send_keys("tomsmith")
        driver.find_element(By.ID, "password").send_keys("SuperSecretPassword!")
        driver.find_element(By.CSS_SELECTOR, "button").click()
        time.sleep(1)
        assert(driver.find_element(By.CSS_SELECTOR, ".flash.success").is_displayed())
        #assert(driver.find_element(By.CSS_SELECTOR, ".flash.successasdf").is_displayed())
